### PR TITLE
Update Connectable Javadoc

### PR DESCRIPTION
**Pull request by sunyal**
- reference UI use case for Android using effects and models

### CODE REVIEW

**Comment by togi** [comment]
Hmm I'm not sure I understand what you're trying to say, maybe you could rephrase it? It's kind of deliberate that the documentation doesn't mention models or android, since the Connectable interface actually is much more general than just that.

**Comment by sunyal** [comment]
@togi I updated the Javadoc to not reference any wiki links for Android related information but I added a comment stating the other scenario.

**Comment by sunyal** [comment]
@togi does this sound better?

 * <p>Alternatively used in {@link MobiusLoop.Controller#connect(Connectable)} to connect the
 * controller to a Connectable.  In that case, the incoming values will be events, and the outgoing
 * values will be models.

File: mobius-core/src/main/java/com/spotify/mobius/Connectable.java [hunk 1]
```diff
@@ -29,6 +30,10 @@
  * Mobius loop. In that case, the incoming values will be effects, and the outgoing values will be
  * events that should be sent back to the loop.
  *
+ * <p>Alternatively used in {@link Controller#connect(Connectable)} to connect a view to the
+ * controller. In that case, the incoming values will be events, and the outgoing values will be
```
Comment (anawara) [inline mobius-core/src/main/java/com/spotify/mobius/Connectable.java]: actually in this case the input `I` is the Model and the output `O` is the Event

Comment (sunyal) [reply mobius-core/src/main/java/com/spotify/mobius/Connectable.java]: @anawara thanks missed that - updated

**Comment by pettermahlen** [comment]
Thanks @sunyal !

**Pull request merged**